kin-sdk/src/androidTest/java/kin/sdk/IntegConsts.java

```diff
@@ -3,7 +3,15 @@
 
 final class IntegConsts {
 
-    static final String TEST_NETWORK_URL = "http://10.0.2.2:8000";
-    static final String TEST_NETWORK_ID = "private testnet";
-    static final String URL_CREATE_ACCOUNT = "http://10.0.2.2:8001/?addr=";
+//    static final String TEST_NETWORK_URL = "http://10.0.2.2:8000";
```

Comment:
let's delete these commented lines and comments and add a task to handle it. (o.w. will meet this "temporary" comment in a year :laughing: )